hummingbot/connector/perpetual_trading.py

```diff
@@ -63,8 +63,7 @@ def funding_payment_span(self) -> List[int]:
     def position_mode(self) -> PositionMode:
         return self._position_mode
 
-    @position_mode.setter
-    def position_mode(self, value: PositionMode):
+    def set_position_mode(self, value: PositionMode):
```

Comment:
Why was the setter annotation removed? Do we want to prevent developers from using this as a property for setting the value?